docs/openapi/components/schemas/common/ShippingInstructionsCertificate.yml

```diff
@@ -0,0 +1,82 @@
+$linkedData:
+  term: ShippingInstructionsCertificate
+  '@id': https://w3id.org/traceability#ShippingInstructionsCertificate
+title: Shipping Instructions Certificate
+description: >- 
+  Shipping Instructions or Shipper's Letter of Instruction is a form issued by a shipper to authorize a carrier to issue a bill of lading or an air waybill on the shipper's behalf. The form contains all details of shipment (e.g., shipper, consignee, bill-to-party, commodity, pieces, weight, cube, etc.) and authorizes the carrier to sign the bill of lading in the name of the shipper.
+  (source: Olegario Llamazares: DICTIONARY OF INTERNATIONAL TRADE, Key definitions of 2000 trade terms and acronyms).
+  Model based on https://service.unece.org/trade/uncefact/publication/Transport%20and%20Logistics/itigg%20unttc/Shipping%20Instructions/Multimodal%20Shipping%20Instructions_D21B/HTML/001.htm.
+type: object
+properties:
+  '@context':
+    type: array
+    const:
+      - 'https://www.w3.org/2018/credentials/v1'
+      - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    const:
+      - VerifiableCredential
+      - ShippingInstructionsCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    type: object
+  proof:
+    type: object
+  relatedLink:
+    title: Related Link
+    description: Links related to this verifiable credential
+    type: array
+    items:
+      $ref: ./LinkRole.yml
+additionalProperties: false
+required: []
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "https://example.com/credential/123",
+    "type": [
+      "VerifiableCredential",
+      "ShippingInstructionsCertificate"
+    ],
+    "name": "Shipping Instructions",
+    "issuanceDate": "2022-03-04T13:40:00Z",
+    "issuer": {
+      "type": "Organization",
+      "id": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+      "name": "Xxinau Manufacturing Co. Ltd.",
+      "description": "Advanced Production - Delivered",
+      "address": {
+        "type": "PostalAddress",
+        "streetAddress": "Xin Fei Da Dao 139",
+        "addressLocality": "Xindao",
+        "addressRegion": "Fujian Province",
+        "postalCode": "361100",
+        "addressCountry": "CN"
+      }
+    },
+    "credentialSubject": {
+      "type": [
```

Comment:
This [link](https://service.unece.org/trade/uncefact/publication/Transport%20and%20Logistics/itigg%20unttc/Shipping%20Instructions/Multimodal%20Shipping%20Instructions_D21B/HTML/001.htm) implies theres two separate pieces, a consignment and an "exchanged document", i think the exchanged document is missing?

Reply:
The example is clearly incomplete, yet its type signals the intent.

I read your comment as suggesting another type here... which type are you suggesting?

Reply:
This is from the link that Nis provided as a source, It outlines two distinct documents if im reading it right a consignment and an exchange document, the properties for both are in that link
![image](https://user-images.githubusercontent.com/19310771/157102735-79307e41-98d2-408b-ba5c-130f6fe748ee.png)

Reply:
@mkhraisha, `Exchanged Document` is "The header document information for a use of this master message assembly" which in our context it the Verifiable Credential. That is how we define aspects such as type, issue date, cancellation date, recipient.